app/presenters/step_by_step_page_presenter.rb

```diff
@@ -0,0 +1,23 @@
+class StepByStepPagePresenter
+  include TimeOptionsHelper
+
+  def initialize(step_by_step_page)
+    @step_by_step_page = step_by_step_page
+  end
+
+  def summary_metadata
```

Comment:
I know that this method was copied wholesale from the view, but I think it could do with a refactor in the next commit to make it more readable.

For example:
an `attr_reader` could be used to access `@step_by_step_page`
the last saved text could be split into a method, 
e.g.
```
def last_saved
  last_saved_time = format_full_date_and_time(step_by_step_page.updated_at)
  return "#{last_saved_time} by #{@step_by_step_page.assigned_to}" if assigned?

  last_saved_time
end

def assigned?
  step_by_step_page.assigned_to.present?
end
```

Then in the `items` hash, set `"Last saved": last_saved`

Reply:
Nice, thanks 👍